app/Kconfig.defaults

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2024 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# HID
+config ZMK_HID_KEYBOARD_REPORT_SIZE
+    default 6
+
+config ZMK_HID_CONSUMER_REPORT_SIZE
+    default 6
+
+# Underglow
+# This default value cuts down on tons of excess .conf files, if you're using GPIO, manually disable this
+config SPI
+    default y
```

Comment:
This will have the effect of unconditionally enabling SPI, AFAICT. This is definitely *not* what we want.

Reply:
Good catch, fixed. It seems I may have been a bit overenthusiastic with the removal of if-blocks, went back over things and added some back where it seemed necessary.